doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -21,7 +21,7 @@ full-featured version of Chapel, refer to
    packages you should have available to build and run Chapel.
 
 
-1) If you don't already have the Chapel 1.25 source release, see
+1) If you don't already have the Chapel 1.25.1 source release, see
```

Comment:
This is another place where I think we normally don't specialize on the '.1', but I also don't
think there's any reason not to (and obviously the file itself has to be updated anyway due
to the filenames that follow).  I guess the dual question is "should we use `1.n.0` for official
releases rather than `1.n`?  (where I think we mostly prefer `1.n` simply because it's simpler
/ easier to read and say.